src/tools/sfdk/command.cpp

```diff
@@ -1660,12 +1660,24 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
     }
 
     if (arguments.first() == "remove") {
-        if (!P::checkPositionalArgumentsCount(arguments, 2, 2))
+        QCommandLineParser parser;
+        QCommandLineOption snapshotsOfOption(QStringList{"snapshots-of"});
+        parser.addOption(snapshotsOfOption);
```

Comment:
I would prefer using the plural version and separate with blank line for consistency :)

```suggestion

        parser.addOptions({snapshotsOfOption});
```

Reply:
Right, now I remember you complained about the same thing the last time I did changes here...